.github/workflows/e2e.yaml

```diff
@@ -90,7 +100,7 @@ jobs:
         with:
           account_id: ${{ vars.ACCOUNT_ID }}
           role: ${{ vars.ROLE_NAME }}
-          region: ${{ vars.AWS_REGION }}
+          region: ${{ vars.PROMETHEUS_REGION }}
```

Comment:
If you are adding this, can you make a note that this needs to be set in the [README](https://github.com/aws/karpenter/blob/main/test/README.md)